lib/config/recommended.js

```diff
@@ -30,6 +32,7 @@ module.exports = {
     'no-unbound': true,
     'no-unnecessary-component-helper': true,
     'no-unused-block-params': true,
+    'no-yield-only': true,
```

Comment:
this one is difficult because the default blueprint for components generates such a template

Reply:
Agree, we shouldn't fail on `ember generate component`'s output...

Reply:
updated to remove this one, the others are good though...